core/unix/loader_linux.c

```diff
@@ -142,8 +147,10 @@ static size_t client_tls_size = 2 * 4096;
  *    0xf779e5bc:  mov    %eax,0x8(%edx)
  *    0xf779e5bf:  movb   $0x0,%gs:0x4fe
  */
-/* On A32, the pthread is put before tcbhead instead tcbhead being part of pthread */
-static size_t tcb_size_estimate = IF_X64_ELSE(0x1000, 0x500);
+/* On AArchXX and RISCV, the pthread struct is put before tcbhead instead of tcbhead
+ * being part of the pthread struct.
+ */
+static size_t tcb_size_estimate = IF_X64_ELSE(0x1000, 0x800);
```

Comment:
Re the s/0x500/0x800/

Is this related to "Removes the TCB padding to correctly work with the new ordering."?